### ISSUE TITLE
xgboost: allow to show feature importances computed by different algorithms

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#154]
Extracted from https://github.com/TeamHG-Memex/eli5/issues/150: it'd be nice to show feature importances computed using all algorithms, e.g. in 3 columns. 

I wonder how this works with https://github.com/TeamHG-Memex/eli5/issues/120 though.

**Comment by lopuhin** [comment]
Some xgboost feature importance libraries suggested @ermakovpetr:
https://github.com/Far0n/xgbfi
https://github.com/limexp/xgbfir
